### ISSUE TITLE
Experiment items truncated after apostrophes

### ISSUE THREAD

**Issue by gouskova** [tlozoot/experigen#4]
Yo--I have a few sentences in my dataset (entirely text-based experiment) that contain apostrophes, like "She likes it when it's raining." The part after the apostrophe is presented on the slide but does not get recorded to the results file, so the .csv shows only "She likes it when it". I am guessing it has to do with apostrophes being treated as quotes...

**Comment by maikombecker** [comment]
Yeah, this is a known bug. One easy thing to do is replace the offending single quote (') with the unicode single quotation mark (’).

**Comment by gouskova** [comment]
That would have helped if I'd known it before I lost a bunch of data points
:) I have a couple of stimuli that are identical up to the apostrophe, and
they are all 'neutralized' beyond salvation now. Oh well.

There must be something that can be done to fix this, though? Like escaping
special characters? Something?

**Comment by maikombecker** [comment]
Sorry... We aren't maintaining this version anymore. I suggest trying this fork:
https://github.com/aquincum/experigen

**Comment by gouskova** [comment]
Hmm, perhaps mention that on the git page as well as here, which still
points to the tlozoot version?

http://becker.phonologist.org/experigen/

**Comment by maikombecker** [comment]
Yes, will do.
